internal/api/users/list_test.go

```diff
@@ -15,32 +15,51 @@ func TestList(test *testing.T) {
 
 	users := make([]*model.ServerUser, 0, len(usersMap))
 	for _, user := range usersMap {
+		if user.GetName(true) == "root" {
```

Comment:
Use `user.Email == model.RootUserEmail`.

Using name is dangerous, since user's can set their own name.
I know this is just testing code, but imagine an attack where the user just needs to set their name to "root" to get privileges.
I think I remember an OSX attack like that a few years back.